### DISCUSSION TITLE
Ticket Delivery, SMS & Immediate options?

### DISCUSSION THREAD

**Discussion by Mcc83** [pretix/pretix#3048] [Q&A]
Hey guys, two-part question.

Has anyone figured out an effective way to deliver tickets via SMS? I deal with a... special group of customers that need their tickets emailed and texted to them (because they cant be bothered to read or type in emails properly),

While the custom ticket design tools are very cool, is there anyway to display the QR code immediately on the Order Confirmed page and skip the download PDF option? 

My client base is pretty much drunk toddlers so I am inclined/motivated to make ticket order delivery excessive and overwhelming from every direction.

**Comment by raphaelm** [comment]
> Has anyone figured out an effective way to deliver tickets via SMS? I deal with a... special group of customers that need their tickets emailed and texted to them (because they cant be bothered to read or type in emails properly),

There is currently no real SMS support in pretix. pretix can collect phone numbers, so it could be built using the REST API or plugin API. There is an (unmaintained, not up to date) plugin at https://github.com/rixx/pretix-twilio that could be used as a start.

At some point, we'll likely integrate this into pretix, but it's not currently on the roadmap specifically.

> While the custom ticket design tools are very cool, is there anyway to display the QR code immediately on the Order Confirmed page and skip the download PDF option?

No, currently not.

**Comment by candidexmedia** [comment]
I was also curious about this, but moreso for texting registrants to remind them of the event, and whether there was a plugin/intergration for that. 

@Mcc83 : What did you end up doing?
